web/js/modules/product-picker/reducers.js

```diff
@@ -166,7 +166,8 @@ export function productPickerReducer(state = productPickerState, action) {
       return {
         ...state,
         categoryType: 'featured',
-        category,
+        // category,
```

Comment:
I would remove this comment at line 169.

Reply:
Removed